homeassistant/components/climate/__init__.py

```diff
@@ -329,19 +333,13 @@ def target_humidity(self) -> int | None:
         return self._attr_target_humidity
 
     @property
-    def hvac_mode(self) -> str:
-        """Return hvac operation ie. heat, cool mode.
-
-        Need to be one of HVAC_MODE_*.
-        """
+    def hvac_mode(self) -> HVACMode | str:
```

Comment:
Should we change the service handler too? It's a bit weird to have different types in the service handler compared to the state property.

Reply:
Yeah agreed